apps/mobile-wallet/src/components/TransactionListItem.tsx

```diff
@@ -57,27 +57,11 @@ const TransactionListItem = memo(({ tx, showInternalInflows = false, ...props }:
         </TransactionIcon>
       }
       rightSideContent={
-        <>
-          <AssetLogos>
-            {assets.map((asset) => (
-              <AssetLogo assetId={asset.id} key={asset.id} size={20} />
-            ))}
-          </AssetLogos>
```

Comment:
This means that unknown tokens are now hidden? That can be confusing when sending unknown tokens cause you'll only see some dust alph in the tx list item...